src/test/java/com/epam/ta/reportportal/util/ItemInfoUtilsTest.java

```diff
@@ -70,35 +69,110 @@ void shouldNotFindAttribute() {
 
   @Test
   void nullAttributeResourceCollectionTest() {
-    Optional<ItemAttributeResource> itemAttributeResource = ItemInfoUtils.extractAttributeResource(
-        null, "key");
+    Optional<ItemAttributeResource> itemAttributeResource = ItemInfoUtils.extractAttributeResource(null, "key");
     assertTrue(itemAttributeResource.isEmpty());
   }
 
   @Test
   void emptyAttributeResourcesCollectionTest() {
-    Optional<ItemAttributeResource> itemAttributeResource = ItemInfoUtils.extractAttributeResource(
-        Collections.emptyList(), "key");
+    Optional<ItemAttributeResource> itemAttributeResource = ItemInfoUtils.extractAttributeResource(Collections.emptyList(), "key");
     assertTrue(itemAttributeResource.isEmpty());
   }
 
   @Test
   void shouldFindAttributeResource() {
     String key = "key1";
-    Optional<ItemAttributeResource> itemAttributeResource = ItemInfoUtils.extractAttributeResource(
-        getAttributeResources(), key);
+    Optional<ItemAttributeResource> itemAttributeResource = ItemInfoUtils.extractAttributeResource(getAttributeResources(), key);
     assertTrue(itemAttributeResource.isPresent());
     assertEquals(key, itemAttributeResource.get().getKey());
   }
 
   @Test
   void shouldNotFindAttributeResource() {
     String key = "not-exist";
-    Optional<ItemAttributeResource> itemAttributeResource = ItemInfoUtils.extractAttributeResource(
-        getAttributeResources(), key);
+    Optional<ItemAttributeResource> itemAttributeResource = ItemInfoUtils.extractAttributeResource(getAttributeResources(), key);
     assertTrue(itemAttributeResource.isEmpty());
   }
 
+  @Test
+  void updateDescriptionTest() {
+    BulkInfoUpdateRQ.Description description = new BulkInfoUpdateRQ.Description();
```

Comment:
Maybe in such cases, when we prepare data, make action and then make assertion, we can use "given-when-then" approach or "arrange-act-assert"


given – an object under test initialization + stubs/mocks creation, stubbing and injection
when – an operation to test in a given test
then – received result assertion + mocks verification (if needed)